### PR TITLE
Firebase - Case Study

### PR DESCRIPTION
**Pull request by AugustWahlberg**
Course Assignment - Development Platforms

### CODE REVIEW

File: firebase-case-study.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+**Appwrite** is also popular BaaS, but distinguish from Firebase in a few ways. Firebase is a cloud-based platform hosted by Google and offers a paid service with a free tier, while Appwrite is a self-hosted platform that is open-source and free to use but requires hosting and setup costs. Firebase has a more straightforward setup process, but provides limited control over your backend infrastructure and data, whereas Appwrite provides greater control but requires more advanced setup. Appwrite also focuses on the APIs. This makes it well-suited for developers who want to build APIs or integrate their applications with other APIs such as server-side validation, image and video manipulation, and more.
```
Comment (Fermain) [inline firebase-case-study.md:75]:
```suggestion
**Appwrite** is also popular BaaS, but is distinguished from Firebase in a few ways. Firebase is a cloud-based platform hosted by Google and offers a paid service with a free tier, while Appwrite is a self-hosted platform that is open-source and free to use but requires hosting and setup costs. Firebase has a more straightforward setup process, but provides limited control over your backend infrastructure and data, whereas Appwrite provides greater control but requires more advanced setup. Appwrite also focuses on the APIs. This makes it well-suited for developers who want to build APIs or integrate their applications with other APIs such as server-side validation, image and video manipulation, and more.
```

File: firebase-case-study.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+
+## Summary
+
+When selecting a BaaS, Firebase may be a great option due to its scalable nature, real-time database capabilities, and ease of authentication and hosting. However, it's important to consider its limitations and how they may impact your project over time. There are many excellent BaaS platforms that compete with Firebase, so it's crucial to thoroughly consider your options before decide which one to choose.
+
+## Prizing
```
Comment (Fermain) [inline firebase-case-study.md:82]:
```suggestion
## Pricing
```

File: firebase-case-study.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+**Bandwidth**: Firebase provides scalable bandwidth for app data transfer, with costs based on data volume and frequency of transfer.
+
+**Cloud Functions**: Firebase provides serverless computing through Cloud Functions, with costs based on the number of invocations and compute time.
+
+**Authentication**: Firebase provides authentication services, with costs based on the number of authentications performed.
+
+**Analytics**: Firebase provides real-time analytics, with costs based on the data processed by the analytics service.
+
+Firebase offers a free tier with limited usage of each feature, and flexible payment options, such as monthly and annual billing. This allows you to get started and evaluate your app's needs without immediate costs. You will find more detailed information here: https://firebase.google.com/pricing
```
Comment (Fermain) [inline firebase-case-study.md:95]:
```suggestion
Firebase offers a free tier with limited usage of each feature, and flexible payment options, such as monthly and annual billing. This allows you to get started and evaluate your app's needs without immediate costs. You will find more detailed information [here](https://firebase.google.com/pricing).
```

**Review by Fermain — CHANGES_REQUESTED** [review]
This is a well written case study with only minor mistakes to correct. It is a good length, provides an interesting overview of this useful platform and uses the correct formatting throughout!

File: firebase-case-study.md (hunk 3 above)
Comment (Fermain) [reply firebase-case-study.md:95]: Please could you pick at least one of the other firebase case studies to peer review.

**Comment by AugustWahlberg** [comment]
Thanks for the helpful feedback!  Added per review to this Firebase case study: https://github.com/NoroffFEU/development-platforms/pull/33

**Review by Fermain — APPROVED** [review]
Feedback is in Moodle, well done and thanks again

**Pull request merged**